src/drivers/default.rs

```diff
@@ -1,30 +1,62 @@
-pub struct Driver {
+use std::result;
+use std::error::Error;
+use std::fmt;
+use driver::Driver;
+
+type Result<T> = result::Result<T, NoError>;
+
+pub struct DefaultDriver {
   target: String,
-  status: isize,
+  status: isize
 }
 
-impl Driver {
+impl DefaultDriver {
```

Comment:
We've had this conversation before and I keep flip flopping 😐

My most recent reasoning is that no matter where in the code the name of the thing should be representative without an alias. This is good for A) re-exporting and B) stack traces/error messages.

In addition I think it saves lines of code. In the file `Driver` needs to be aliased as `DriverTrait` and in other files (if ambiguous) `Driver` must be aliased as `DefaultDriver`. I think this is kinda comparable to Java package names compared to class names.